src/main/java/seedu/address/logic/parser/CliSyntax.java

```diff
@@ -12,4 +12,11 @@
     public static final Prefix PREFIX_ADDRESS = new Prefix("a/");
     public static final Prefix PREFIX_TAG = new Prefix("t/");
 
+    public static final Prefix PREFIX_TASK = new Prefix("t/");
```

Comment:
Prefix clashes with `TAG`, does it matter though?

Reply:
changed tag to tt/